sections/loans/components/ActionBox/ActiveBorrowsTab/ModifyLoanMenu.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import { useRouter } from 'next/router';
+import styled from 'styled-components';
+import OutsideClickHandler from 'react-outside-click-handler';
+import { zIndex } from 'constants/ui';
+import { Loan } from 'queries/loans/types';
+import { LOAN_TYPE_ETH } from 'sections/loans/constants';
+
+const MODAL_WIDTH: number = 105;
+const MODAL_TOP_PADDING: number = 15;
+
+type BorrowModifyModalProps = {
+	actions: Array<string>;
+	loan: Loan;
+};
+
+const BorrowModifyModal: React.FC<BorrowModifyModalProps> = ({ actions, loan }) => {
+	const router = useRouter();
+
+	const buttonRef = React.useRef<HTMLDivElement>(null);
+	const [{ top, left }, setPosition] = React.useState<any>({});
+	const [modalContainer, setModalContainer] = React.useState<HTMLElement | null>(null);
+
+	const open = top && left;
+
+	const onOpen = () => {
+		const el: HTMLElement = buttonRef.current!;
+		const elRect = el.getBoundingClientRect();
+		const left = elRect.left - window.scrollX - 20;
+		const top = elRect.top + window.scrollY + MODAL_TOP_PADDING;
+		setPosition({ top, left });
+	};
+	const onClose = () => setPosition({});
+
+	const onStartModify = (action: string) => {
+		router.push(`/loans/${loan.type === LOAN_TYPE_ETH ? 'eth' : 'erc20'}/${loan.id}/${action}`);
+	};
+
+	React.useEffect(() => {
+		if (typeof window !== 'undefined') {
+			setModalContainer(document.getElementById('modal-container'));
+		}
+	}, []);
+
+	return (
+		<Menu>
+			<Button onClick={onOpen} ref={buttonRef}>
+				ACTIONS{' '}
```

Comment:
Probably needs to be in the translation file

Reply:
Done.